core/common/src/DayOfWeek.kt

```diff
@@ -25,8 +23,7 @@ public expect enum class DayOfWeek {
  */
 public val DayOfWeek.isoDayNumber: Int get() = ordinal + 1
 
-@SharedImmutable
-private val allDaysOfWeek = DayOfWeek.values().asList()
+private val allDaysOfWeek = DayOfWeek.entries
```

Comment:
Please revert enum entires from this PR, it is fixed more thoroughly in #319

Reply:
Sure, I'll have to do that while rebasing on top of #319 anyway.